README.md

```diff
@@ -50,10 +50,20 @@ $ npm install download-chromium
 
 ## API
 
-### download({ platform = currentPlatform, revision = '499413', log = false })
-
+### download
+```
+download({ 
+	platform = currentPlatform, 
+	revision = '499413', 
+	log = false, 
+	onProgress = undefined, 
+	installPath = '{root folder od download-chromium module}' })
```

Comment:
```suggestion
	installPath = rootFolderOfThisModule
})
```

Reply:
done, following other comment I modified it to `'{rootFolderOfThisModule}/.local-chromium'`